Add placeholder gears as logo

lilyminium opened the pull request:
Adds the logo to MDAKits

Changes made in this Pull Request:
 - 


PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG.md updated?
 - [ ] AUTHORS.md updated?
 - [ ] Issue raised/referenced?

pep8speaks commented:
Hello @lilyminium! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`tests/test_options.py`](https://github.com/MDAnalysis/cookiecutter-mdakit/blob/b0d345d4a180f1f8503543741fb1150a58c62723/tests/test_options.py):

> [Line 57:80](https://github.com/MDAnalysis/cookiecutter-mdakit/blob/b0d345d4a180f1f8503543741fb1150a58c62723/tests/test_options.py#L57): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (83 > 79 characters)

lilyminium commented:
<img width="780" alt="Screenshot 2024-06-22 at 11 12 17 pm" src="https://github.com/MDAnalysis/cookiecutter-mdakit/assets/31115101/57dd7e25-8290-429e-b0a0-5c210a32076b">

Added some docs explaining to edit the logo, and also updated MDAKits placeholder to this.

lilyminium commented:
@orbeckst is this what you were thinking? I could also drop the question mark in the placeholder if that's simpler!

orbeckst reviewed (approved):
Looking great. I’d leave the template with question mark in, just as you have it. 

Once you’re done with the logo here, can you also add the official MDAkit logo to the branding repo?

(I’m not 100% sure if we should put the empty template into branding as well — it might become confusing. Perhaps have a separate repo for free templates???)

IAlibay commented on tests/test_options.py:
```diff
@@ -39,8 +39,8 @@ def test_official_mda_theme(self, tmpdir):
             assert '"mda_official": True,' in text
-            assert "html_logo" not in text
-            assert "html_favicon" not in text
+            # assert "html_logo" not in text
```
maybe worth removing these if they are no longer useful?

lilyminium replied:
Thank you -- done!

IAlibay reviewed (approved):
Approving, although with the one comment re: removing commented out test lines.
